### ISSUE TITLE
bug in concatenation layer

### ISSUE THREAD

**Issue by AroMorin** [keras-team/keras#7783]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

running this:
	resnet = resnet.get_layer('activation_49').output
	flatten = Flatten()(resnet)
	seed = Input(shape=(7,))

	merged = concatenate([flatten, seed], axis=-1)

results in : zero-dimensional arrays cannot be concatenated

However, running this:
	resnet = resnet.get_layer('activation_49').output
	flatten = Flatten()(resnet)
	seed = Input(shape=(7,))

	merged = merge([flatten, seed], mode='concat', concat_axis=-1)

compiles well without errors! Bug?

**Comment by facaiy** [comment]
Could you give an executable (self-contained) code?

**Comment by AroMorin** [comment]
Are you unable to reproduce the behavior using the outlined code?

**Comment by black-sither** [comment]
I had the same problem..it turned out that there was some trouble recognizing concatenate to be associated with keras for some reason..

doing this solved the issue for me

`from keras.layers.merge import concatenate as cnct`
`merged = cnct([flatten,seed])`
